products/compute/api.yaml

```diff
@@ -3084,6 +3084,23 @@ objects:
           character, which cannot be a dash.
         input: true
         required: true
+      - !ruby/object:Api::Type::Enum
+        name: 'quicOverride'
+        description: |
+          Specifies the QUIC override policy for this resource. This determines
+          whether the load balancer will attempt to negotiate QUIC with clients
+          or not. Can specify one of NONE, ENABLE, or DISABLE. Specify ENABLE to
+          always enable QUIC, Enables QUIC when set to ENABLE, and disables QUIC
+          when set to DISABLE. If NONE is specified, uses the QUIC policy with
```

Comment:
This sentence doesn't quite scan.  I've filed a docs bug, but can you rephrase?

Reply:
Done